Jenkinsfile

```diff
@@ -0,0 +1,22 @@
+pipeline {
+    agent any
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr:'10'))
+        timeout(time: 15, unit: 'MINUTES')
+        ansiColor('xterm')
+    }
+    
+    stages {
+        stage('Build Image') {
+            steps {
+                script {
+                    sh "ls"
```

Comment:
@ashish1605 Being new to jenkins, I may be missing something obvious. What purpose does the `sh "ls"` serve?

Reply:
It is just a `list` command I gave for test purpose. We can give any command here but I just added list command for testing purpose!!